src/app/zap-templates/zcl/data-model/chip/tls-certificate-management-cluster.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/tls/TLSCertificateManagement.adoc
+Parameters: in-progress 
+Git: 1.4-333-gfa9c78773
+-->
+<configurator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../zcl.xsd">
+  <domain name="General"/>
+  <struct name="TLSCertStruct" apiMaturity="provisional">
+    <cluster code="0x0801"/>
+    <item fieldId="0" name="CAID" type="int16u"/>
+    <item fieldId="1" name="Certificate" type="long_octet_string" length="3000"/>
+  </struct>
+
+  <struct name="TLSClientCertificateDetailStruct" apiMaturity="provisional">
+    <cluster code="0x0801"/>
+    <item fieldId="0" name="CCDID" type="int16u"/>
+    <item fieldId="1" name="ClientCertificate" type="long_octet_string" length="3000"/>
+    <item fieldId="2" name="IntermediateCerts" array="true" type="octet_string"/>
```

Comment:
Spec comment: why is "Certificate" abbreviated in one field name but not the other?  Seems like a weird inconsistency.  Please make sure a spec issue is filed.

Reply:
Will rename to IntermediateCertificates in spec.